components/feature/search/src/main/assets/search/list.json

```diff
@@ -240,6 +240,11 @@
         "visibleDefaultEngines": [
           "google-b-m", "bing", "ddg", "wikipedia-es"
         ]
+      },
+      "experimental-hidden": {
+        "visibleDefaultEngines": [
+          "amazon-es"
+        ]
```

Comment:
I don't think we read anything from the `experimental-hidden` section? Afaik, this needs to go into the section above?

Reply:
This one I'm not sure.  I added this to experimental-hidden because I don't know if it's common for countries outside of Spain to use es-ES as their primary language.  But you're right, it could be better if I just add it to the the visible section and see if there's any user complaints.  I'll move it thanks.

Reply:
I've tested it and looks like only if the user chooses Espanol (Espana) Amazon.es shows up as a search engine.  I think this is working as expected.  Thanks